python/tvm/relay/frontend/onnx.py

```diff
@@ -3202,6 +3202,24 @@ def from_onnx(self, graph, opset, get_output_expr=False):
                 outputs_num = 1
             else:
                 outputs_num = len(op)
+            if outputs_num > 1:
+                valid_outputs = [False] * outputs_num
+                for i in range(len(node_output)):
+                    if node_output[i] != "":
+                        valid_outputs[i] = True
+                if not all(valid_outputs):
+                    tup = op.astuple()
+                    if isinstance(tup, _expr.Tuple):
```

Comment:
Does `op.astuple` not always return an `_expr.Tuple`?

Reply:
No, we use TupleWrapper for other nodes, like call nodes with multiple outputs, see split: https://github.com/apache/tvm/blob/461d06eb5cfc7954f1983779acd05c47cea269f1/python/tvm/relay/op/transform.py#L861